mirascope/core/vertex/call_response_chunk.py

```diff
@@ -0,0 +1,82 @@
+"""This module contains the `VertexCallResponseChunk` class.
+
+usage docs: learn/streams.md#handling-streamed-responses
+"""
+
+from vertexai.generative_models import FinishReason, GenerationResponse
+
+from ..base import BaseCallResponseChunk
+
+
+class VertexCallResponseChunk(
+    BaseCallResponseChunk[GenerationResponse, FinishReason]  # , Candidate.FinishReason]
+):
+    """Convenience wrapper around chat completion streaming chunks.
+
+    When using Mirascope's convenience wrappers to interact with Vertex models via
+    `VertexCall`, responses using `VertexCall.stream()` will return a
```

Comment:
same here, this needs to be updated, please reference `openai` and also update any other modules that have this issue

Reply:
Fixed